content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -228,12 +228,6 @@ Cluster DNS (CoreDNS) will not start up before a network is installed.**
 
 {{< /caution >}}
 
-{{< note >}}
-Currently Calico is the only CNI plugin that the kubeadm project performs e2e tests against.
-If you find an issue related to a CNI plugin you should log a ticket in its respective issue
-tracker instead of the kubeadm or kubernetes issue trackers.
```

Comment:
I would not remove the paragraph entirely, just the line 
```
Currently Calico is the only CNI plugin that the kubeadm project performs e2e tests against.
```

Reply:
yes, it makes sense to keep this note:
> If you find an issue related to a CNI plugin you should log a ticket in its respective issue
tracker instead of the kubeadm or kubernetes issue trackers.

we can omit talking about our e2e.